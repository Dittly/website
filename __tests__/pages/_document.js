/* eslint-env jest */
jest.mock('react-ga')
jest.mock('next/document')

import React from 'react'
import Document from '../../pages/_document'
import renderer from 'react-test-renderer'

it('renders / correctly', () => {
  const tree = renderer.create(
    <Document />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
