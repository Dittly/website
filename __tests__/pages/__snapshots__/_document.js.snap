// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders / correctly 1`] = `
<html>
  <body>
    <div
      className="root"
    />
  </body>
</html>
`;
